crates/democracy/src/lib.rs

```diff
@@ -577,6 +577,7 @@ pub mod pallet {
         ///
         /// Weight: `O(p)` where `p = PublicProps::<T>::decode_len()`
         #[pallet::weight(T::WeightInfo::cancel_proposal(T::MaxProposals::get()))]
+        // TODO: remove all sudo calls? e.g. cancel_proposal
```

Comment:
We would keep these functions for governance proposals - they would dispatch as root.

Reply:
How do we ensure that the root account can no longer call them?

Reply:
We will remove the sudo pallet if that's what you mean?